source/opt/module.h

```diff
@@ -119,6 +120,10 @@ class Module {
   // Appends a function to this module.
   inline void AddFunction(std::unique_ptr<Function> f);
 
+  // Get the existing DebugInfoNone. If it is null, create one and keep it.
+  Instruction* GetDebugInfoNone(uint32_t type_id, uint32_t result_id,
+                                uint32_t ext_set);
+
```

Comment:
This does not belong on the module, move it to the IRContext.  Also it should not take a result id since it may not create a new instruction.  It should take the next id if it needs to create a new instruction.

Reply:
done